doozer/doozerlib/backend/konflux_fbc.py

```
@@ -781,6 +1092,21 @@ def _update_channel(channel: Dict):
                 # become the only member of new-entry's `skip`.
                 skips = {channel['entries'][0]['name']}
 
+            # Add assembly bundle to skips from the latest named assembly's shipment
+            if assembly_csv_info:
+                if skips is None:
+                    skips = set()
+                if assembly_csv_info.csv_name == olm_bundle_name:
+                    logger.info("Assembly bundle is the same as the current bundle, skipping")
+                    continue
```

Ruff (0.14.14)

[error] `continue` not properly in loop

(F702)